package.json

```diff
@@ -1,56 +1,30 @@
 {
-  "name": "history",
-  "version": "4.10.1",
-  "description": "Manage session history with JavaScript",
-  "repository": "ReactTraining/history",
-  "license": "MIT",
-  "author": "Michael Jackson",
-  "main": "index.js",
-  "module": "esm/history.js",
-  "unpkg": "umd/history.js",
-  "files": [
-    "DOMUtils.js",
-    "ExecutionEnvironment.js",
-    "LocationUtils.js",
-    "PathUtils.js",
-    "cjs",
-    "createBrowserHistory.js",
-    "createHashHistory.js",
-    "createMemoryHistory.js",
-    "createTransitionManager.js",
-    "es",
-    "esm",
-    "umd",
-    "warnAboutDeprecatedCJSRequire.js"
-  ],
-  "sideEffects": false,
+  "name": "history-packages",
+  "private": true,
   "scripts": {
-    "build": "rollup -c",
+    "build": "node ./scripts/build.js",
+    "size": "filesize",
     "clean": "git clean -fdX .",
-    "lint": "eslint modules",
+    "lint": "eslint .",
     "prepublishOnly": "yarn build",
```

Comment:
is this prepublishOnly needed here? seems not necessary for a private monorepo root